### PR TITLE
Add Graphite output

### PR DESCRIPTION
**Pull request by titilambert**
This is a new PR of #451 rebased on 0.3.0

### CODE REVIEW

File: plugins/outputs/graphite/graphite.go [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+	for _, point := range points {
+		// Get name
+		name := point.Name()
+		// Convert UnixNano to Unix timestamps
+		timestamp := point.UnixNano() / 1000000000
+
+		for field_name, value := range point.Fields() {
+			// Convert value
+			var value_str string
+			switch value := value.(type) {
```
Comment (sparrc) [inline plugins/outputs/graphite/graphite.go]: You don't need to do this type switch. Just do:

`value_str := fmt.Sprintf("%v", value)`

File: plugins/outputs/graphite/graphite.go [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+	Timeout int
+	conns   []net.Conn
+}
+
+var sampleConfig = `
+  # TCP raw endpoint for your graphite instance.
+  servers = ["mygraphiteserver:2003"] # default "localhost:2003"
+  # Prefix metrics name
+  prefix = "" # default ""
+  # Timeout in second
```
Comment (sparrc) [inline plugins/outputs/graphite/graphite.go]: Can you describe the timeout better?

**Comment by sparrc** [comment]
looks good, I made a few comments, you can also just have this PR merge back into master now. Thanks for dealing with all the recent changes!

**Comment by titilambert** [comment]
@sparrc Hello ! I fixed all your comments and rebased on master.

**Comment by sparrc** [comment]
Awesome, thanks a bunch @titilambert! :+1:

**Pull request closed without merging**